ui/admin/tests/acceptance/auth-methods/update-test.js

```diff
@@ -115,21 +132,28 @@ module('Acceptance | auth methods | update', function (hooks) {
     await select('form fieldset:nth-of-type(1) select', 'RS384');
     await click('form fieldset:nth-of-type(1) [title="Add"]');
     // Remove all allowed audiences
-    await Promise.all(
-      findAll('form fieldset:nth-of-type(2) [title="Remove"]').map((element) =>
-        click(element)
-      )
+    const allowedAudiencesList = await Promise.all(
```

Comment:
We don't need to be awaiting or using promise.all since we're just finding html elements